Diffing_Engine/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="CompareNETObjects" version="4.66.0" targetFramework="net452" />
+  <package id="CompareNETObjects" version="4.67.0" targetFramework="net452" />
```

Comment:
No problem with the upgrade of Kellerman Software to version 4.67. I have tested it.